etip/trackers/models.py

```diff
@@ -81,24 +81,38 @@ def clean_fields(self, exclude=None):
                 {'code_signature': "Must be a valid regex."}
             )
 
+    def any_signature_collision(self):
```

Comment:
This is not only about your code changes, but I think we could use this occasion to rename our functions to something more explicit.

I usually like functions starting with "get_" when it returns values and "has_" or "is_" when it returns a boolean. I think here `code_signature_collision` could be renamed to `get_code_signature_collisions` for instance. WDYT?

Reply:
ahah, of course we can go with it. That's just because in Ruby prefixing is a bad naming convention :) I'll rename these methods.

What about 

- `has_any_signature_collision` instead of `any_signature_collision`
- `get_trackers_with_code_signature_collision` instead of `code_signature_collision` 
- `get_trackers_with_network_signature_collision` instead of `network_signature_collision` 
- `__has_same_network_signature` instead of `__same_network_signature`
- `__has_same_code_signature` instead of `__same_code_signature`

What do you think ?

Reply:
Actually it's debatable. **The only thing specified in PEP8 is to use lowercase words separated by underscore** to try to improve readability https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables (note that PEP8 says to use only 1 underscore before private functions and not 2).

Reply:
My linter didn't point me out this >< I can change it too. 

May be, it's larger discussion we can have about conventions we want for this repository (defined rules can go on the contributing file for example). I think it can be a good thing to have such conventions as it helps to understand the code.

Reply:
I've do the changes :)